media/windows/ricoh-printer/ricoh_db_create.py

```diff
@@ -0,0 +1,121 @@
+#!/user/bin/env python3
+
+# Script which runs daily to update staff print totals
+# based on CSVs generated by the printer itself.
+
+import sys
+sys.path.insert(1, '../../python/')
+
+import os
+import re
+import csv
+import datetime
+import sqlite3
+from sqlite3 import Error
+
+#import ECC
+#import PDSChurch
+
+from pprint import pprint
+from pprint import pformat
+
+###########################################################
+
+ricoh_db = "media/windows/ricoh-printer/ricoh.db"
+today = datetime.date.today()
+datestring = f'{today.year}{today.month}{today.day}'
+debug_datestring = '20201013'
+
+def _strip(value):
+    #The CSV puts brackets around values, which we don't want
+    return value.strip('[').strip(']')    
+
+def load_latest_csv():
+    
+    filename = f'RICOH IM C4500_usercounter_{datestring}.csv'
+    
+    debug_filename = 'media/windows/ricoh-printer/RICOH IM C4500_usercounter_20201013.csv'
+    
+    csv_rows = list()
+
+    with open(debug_filename, encoding='utf-8') as csvfile: #TODO: change to filename when retrieval is automated
+        csvreader = csv.DictReader(csvfile)
+        for row in csvreader:
+            csv_rows.append(row)
+
+    print(f"== Loaded {len(csv_rows)} staffers from latest CSV")
```

Comment:
Better to use the ECC logging everywhere (vs. `print()`).  Remember: this script will ultimately run in an automated fashion.  So we'll want it to log to a file so that if (when) anything goes wrong, we can go back and look at the logs to see what happened.

You use the logging functions just like `print()`, but you carry additional semantic information with your choice of function:

* `log.info(...)`: regular ol' info
* `log.debug(...)`: stuff that only you, the developer, need to see while developing/debugging.  It's not something we need to see when we run the program "for real".  Hence, you can have lots of debugging output in the code, but it won't appear unless you create the `log` object with `debug=true`.  It's a handy way to enable/disable a whole swath of output.
* `log.error(...)`: use this when an error occurs (e.g., in the `except` clauses and whatnot).